### PR TITLE
Check out latest release tag before building docs.

### PR DESCRIPTION
**Pull request by twiecki**
Closes https://github.com/pymc-devs/pymc3/issues/3157

### CODE REVIEW

File: build_and_deploy_docs.sh [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 
+latesttag=$(git describe --tags `git rev-list --tags --max-count=1`)
```
Comment (fonnesbeck) [inline build_and_deploy_docs.sh:3]: Nice!

**Comment by fonnesbeck** [comment]
LGTM

**Comment by twiecki** [comment]
OK, docs should be on 3.5.

**Comment by ColCarroll** [comment]
I don't think this actually worked - I can't find where this script is used - it looks like `travis-sphinx` is actually used everywhere.

**Comment by twiecki** [comment]
Oh, I forgot that we switched to that. It seems like `travis-sphinx` can't do that. Maybe we remove travis-sphinx and go back to the manual deployment as we now will only update docs every new release?

**Comment by ColCarroll** [comment]
That's a reasonable idea.

If we have `docs.pymc.io` can we keep `travis-sphinx` deploying to something like `master.pymc.io` or `new.pymc.io`?

**Comment by twiecki** [comment]
I like that! `docs-master.pymc.io`?

**Comment by twiecki** [comment]
Actually that doesn't work, as we can only have one website in gh-pages.

**Pull request merged**